src/main/resources/ath-container/Dockerfile

```diff
@@ -55,40 +56,46 @@ ARG GECKODRIVER_VERSION=0.34.0
 RUN arch=$(arch | sed s/aarch64/arm64/ | sed s/x86_64/amd64/) && \
   if [ "$arch" = "arm64" ] ; \
   then \
-    curl -fsSLO https://github.com/mozilla/geckodriver/releases/download/v${GECKODRIVER_VERSION}/geckodriver-v${GECKODRIVER_VERSION}-linux-aarch64.tar.gz && \
-    tar -xvzf geckodriver-v${GECKODRIVER_VERSION}-linux-aarch64.tar.gz -C /usr/local/bin; \
+    curl -fsSLO "https://github.com/mozilla/geckodriver/releases/download/v${GECKODRIVER_VERSION}/geckodriver-v${GECKODRIVER_VERSION}-linux-aarch64.tar.gz" && \
+    tar -xvzf "geckodriver-v${GECKODRIVER_VERSION}-linux-aarch64.tar.gz" -C /usr/local/bin; \
   else \
-    curl -fsSLO https://github.com/mozilla/geckodriver/releases/download/v${GECKODRIVER_VERSION}/geckodriver-v${GECKODRIVER_VERSION}-linux64.tar.gz && \
-    tar -xvzf geckodriver-v${GECKODRIVER_VERSION}-linux64.tar.gz -C /usr/local/bin ; \
+    curl -fsSLO "https://github.com/mozilla/geckodriver/releases/download/v${GECKODRIVER_VERSION}/geckodriver-v${GECKODRIVER_VERSION}-linux64.tar.gz" && \
+    tar -xvzf "geckodriver-v${GECKODRIVER_VERSION}-linux64.tar.gz" -C /usr/local/bin ; \
   fi
 
 # Maven in repo is not new enough for ATH
 ARG MAVEN_VERSION=3.9.8
-RUN curl -ffSLO https://archive.apache.org/dist/maven/maven-3/"${MAVEN_VERSION}"/binaries/apache-maven-"${MAVEN_VERSION}"-bin.tar.gz && \
-    tar -xvzf apache-maven-"${MAVEN_VERSION}"-bin.tar.gz -C /opt/ && \
-    mv /opt/apache-maven-* /opt/maven
-ENV PATH="$PATH:/opt/maven/bin"
+RUN curl -fsSLO "https://archive.apache.org/dist/maven/maven-3/${MAVEN_VERSION}/binaries/apache-maven-${MAVEN_VERSION}-bin.tar.gz" && \
+    tar -xvzf "apache-maven-${MAVEN_VERSION}-bin.tar.gz" -C /opt/ && \
+    mv "/opt/apache-maven-${MAVEN_VERSION}" /opt/maven
+ENV PATH "$PATH:/opt/maven/bin"
 
 ENV SHARED_DOCKER_SERVICE true
 
+COPY run.sh /usr/bin
+COPY set-java.sh /usr/bin
+COPY vnc.sh /usr/bin
```

Comment:
Not new code, but moved from the bottom of this file so we don't have to keep switching back and forth between users.